packages/react/src/components/MenuButton/MenuButton.tsx

```diff
@@ -157,6 +161,10 @@ export default class MenuButton extends AutoControlledComponent<MenuButtonProps,
     }
   }
 
+  componentDidMount() {
+    handleRef(this.props.componentRef, this)
```

Comment:
You don't need this, refs to class components defaults to classes

```jsx
<Menu ref /> // === <Menu componentRef />
```

Reply:
addressed